WebAssets/package.json

```diff
@@ -59,13 +59,13 @@
         "@typescript-eslint/parser": "^2.26.0",
         "babel-plugin-add-import-extension": "^1.3.1",
         "eslint": "^6.8.0",
-        "execa": "^4.0.0",
         "fast-glob": "^3.2.2",
         "fs-jetpack": "^2.1.0",
         "jest": "^25.1.0",
         "keysim": "^2.0.0",
-        "oldowan": "^1.0.0-preview-2",
+        "oldowan": "^1.0.0-preview-3",
         "ts-jest": "^25.2.1",
+        "ts-node": "^8.9.0",
```

Comment:
where do you keep finding libraries like this

Reply:
Not sure if I like it

Reply:
I don't see an alternative as deno can't be installed through npm?

Reply:
actually would be nice if ms shipped this themselves